js/apps/admin-ui/src/realm-settings/GeneralTab.tsx

```diff
@@ -95,7 +95,9 @@ function RealmSettingsGeneralTabForm({
   save,
   userProfileConfig,
 }: RealmSettingsGeneralTabFormProps) {
-  const { adminClient } = useAdminClient();
+  const {
+    environment: { serverBaseUrl },
+  } = useEnvironment();
```

Comment:
Why does this need to use the `useEnvironment()` hook? The environment passed into it is the same environment that can be statically referenced from `environment.ts`.

Reply:
you need to be able to change the environment when you use this screen to build your own admin ui